syntax = "proto3";

package autodrone;

import "google/protobuf/empty.proto";

// The greeting service definition.
service Drone {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
  rpc GetCamera (google.protobuf.Empty) returns (CameraReply) {}
}

// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloReply {
  string message = 1;
}

message CameraReply {
  string message = 1;
}